sdk/formrecognizer/azure-ai-formrecognizer/tests/recordings/test_to_dict.test_custom_form_labeled_to_dict.yaml

```diff
@@ -0,0 +1,299 @@
+interactions:
```

Comment:
let's delete this since we deleted the test

Reply:
Done